src/explore.sh

```diff
@@ -90,3 +98,18 @@ function explore_files_under_git()
 
   cmd_manager "$flag" "git grep -e \"$regex\" -nI $path"
 }
+
+function explore_files_using_grep()
+{
+  local regex="$1"
+  local path="$2"
+  local flag="$3"
+
+  # Silent by default
+  flag=${flag:-"SILENT"}
+
+  # If user only set regex value
+  path=${path:-"."}
+
+  cmd_manager "$flag" "grep -nrwI $path -e \"$regex\""
```

Comment:
Since you decided to use `cmd_manager` you can use the flag `TEST_MODE` for implementing a new test that validate the command correctness. Take a look at `mk_test`, test `mk_kernel_uninstall_Test` for reference.